pkg/docker/create_devcontainer.go

```diff
@@ -151,30 +151,30 @@ func (d *DockerClient) CreateFromDevcontainer(opts CreateDevcontainerOptions) (s
 				return "", "", err
 			}
 		} else {
-			composeFilePath = filepath.Join(opts.ProjectDir, filepath.Dir(opts.BuildConfig.Devcontainer.FilePath), composeFilePath)
+			composeFilePath = filepath.Join(opts.WorkspaceDir, filepath.Dir(opts.BuildConfig.Devcontainer.FilePath), composeFilePath)
 		}
 
 		options, err := cli.NewProjectOptions([]string{composeFilePath}, cli.WithOsEnv, cli.WithDotEnv)
 		if err != nil {
 			return "", "", err
 		}
 
-		project, err := cli.ProjectFromOptions(ctx, options)
+		workspace, err := cli.ProjectFromOptions(ctx, options)
```

Comment:
This should be reverted back to `project` because it references a docker compose project. It's not a Daytona structure.